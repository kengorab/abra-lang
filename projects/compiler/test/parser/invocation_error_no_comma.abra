a.b(1 abc)