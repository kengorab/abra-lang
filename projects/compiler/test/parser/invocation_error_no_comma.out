Error at %FILE_NAME%:1:7
Unexpected token 'identifier', expected one of ',', ')':
  |  a.b(1 abc)
           ^